Makefile

```
@@ -420,19 +428,15 @@ terratest/test:
 	cd $(TERRAFORM_TEST_DIR) && go test -count=1 -timeout $(TIMEOUT)
 	@ echo "[$@]: Completed successfully!"
 
-TERRAFORM_PYTEST_ARGS ?=
 TERRAFORM_PYTEST_DIR ?= $(TARDIGRADE_CI_PATH)/tests/terraform_pytest
-terraform/pytest: | guard/program/terraform guard/program/pytest guard/python_pkg/tftest
-	@ echo "[$@] Starting Terraform integration test"
-	pytest -v $(TERRAFORM_PYTEST_DIR) $(TERRAFORM_PYTEST_ARGS)
-	@ echo "[$@]: Completed successfully!"
+terraform/pytest: | guard/program/terraform guard/python_pkg/tftest pytest/$(TERRAFORM_PYTEST_DIR)
```

Very nice! I suppose if you wanted to keep using `pytest -v` with this target, you could default PYTEST_ARGS?

```suggestion
terraform/pytest: export PYTEST_ARGS ?= -v
terraform/pytest: | guard/program/terraform guard/python_pkg/tftest
terraform/pytest: | pytest/$(TERRAFORM_PYTEST_DIR)
```

---

I went back and forth on whether to use the "-v" or not.  The default should be no args with the intent that if you want an option, you add it.  And it depends whether you just want a pass/fail indication for tests, or more detail if there is a failure.  If I had used PYTEST_ARGS and set it for all the repos I worked on so far, it would be an easy decision to leave this blank.  I dunno.

---

Roger. Agree. Only mentioned it because this particular target was already using `-v` before, but we don't have to continue that.